lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -169,6 +169,7 @@ export default TestModule.extend({
       Ember.run(function() {
         Ember.setProperties(context, hash);
       });
+      return hash;
```

Comment:
@rwjblue depending on which version of Ember you're using, `Ember.setProperties` might return either `context` or `hash`. I think it makes more sense for it to return `hash` so I standardized around that. I can change it if you feel the other behavior is better.